### DISCUSSION TITLE
Finding paths with 0ns delay

### DISCUSSION THREAD

**Discussion by chrpapadopoulos** [NVlabs/sionna#638] [Q&A] [answered]
Hello everyone!

I have built an indoor scene using blender and using it to find the coverage map and paths between a single Tx and Rx. The wierd thing is that sionna always comes up with one path that has 0ns delay. I don't know how this can be a correct value. Am I missing something?

![image](https://github.com/user-attachments/assets/5cd15329-ead5-4599-a38e-9ee33d544ee5)

![image](https://github.com/user-attachments/assets/2ecc818a-b77e-48f4-8b3e-5e1e96ea48b2)

![image](https://github.com/user-attachments/assets/03ab8019-e7e7-4b79-b389-ef7f7d99994c)

This happens even if I move the Rx to another position as well.
Is this something due to the machine floating point precision? Or maybe something wrong with the scene I have built?

Or maybe it's related to this discussion? https://github.com/NVlabs/sionna/discussions/626

I'm using v0.18.0 at the moment.

Do tell if you need more details on code or scene.
Thank you!

**Comment by SebastianCa** [comment, marked as answer]
Hi @chrpapadopoulos,
per default Sionna normalizes the path delays such that the first path arrives at time t=0 (see [RT tutorial](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#From-Paths-to-Channel-Impulse-Responses) for an example). This behavior can be controlled via the [normalize_delay](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Paths.normalize_delays) property of `Paths`. 
I hope this answers your question.

**Comment by chrpapadopoulos** [comment]
Yes indeed that is what I missed. I can set it to false to get the absolute values if I need to.

Thank you very much!
